core/src/main/scala/kafka/server/KafkaRequestHandler.scala

```diff
@@ -318,9 +345,12 @@ class BrokerTopicStats {
       metrics.close()
   }
 
-  def updateBytesOut(topic: String, isFollower: Boolean, value: Long): Unit = {
+  def updateBytesOut(topic: String, isFollower: Boolean, isReassignment: Boolean, value: Long): Unit = {
     if (isFollower) {
-      updateReplicationBytesOut(value)
+      if (isReassignment)
+        updateReassignmentBytesOut(value)
+      else
+        updateReplicationBytesOut(value)
```

Comment:
I think what we decided in the discussion is to count reassignment bytes toward total replication.

Reply:
Ok, I'll revert this change and we can discuss it when we change throttling.